aiida_quantumespresso/calculations/pw.py

```diff
@@ -145,6 +145,11 @@ def define(cls, spec):
             message='The electronic minimization cycle did not reach self-consistency.')
         spec.exit_code(541, 'ERROR_SYMMETRY_NON_ORTHOGONAL_OPERATION',
             message='The variable cell optimization broke the symmetry of the k-points.')
+
+        # Strong warnings about calculation results, but something tells us that you're ok with that
+        spec.exit_code(610, 'WARNING_ELECTRONIC_CONVERGENCE_NOT_REACHED',
```

Comment:
See comment on the issue
```suggestion
        spec.exit_code(710, 'WARNING_ELECTRONIC_CONVERGENCE_NOT_REACHED',
```